index.html

```diff
@@ -31,13 +32,17 @@ <h1>Learn the Best Ways for Staying Productive</h1>
             <img src="assets/btnImage.png" alt="Icon">Learn More
           </button>
         </div>
-      </article>
-      <article class="right">
-        <img src="assets/headerImage.png" alt="Header Image">
-      </article>
+      </div>
+      <figure class="right">
+        <div>
+          <img src="assets/headerImage.png" alt="Header Image">
+          <hr>          
+        </div>
+
+      </figure>
     </section>
     
-    <section class="last-section">
+    <section class="features-section">

```

Comment:
Use a CSS pseudo-element to add the colored dot on each feature card icon, consistent with the design.